python/lsst/pipe/tasks/subtractBrightStars.py

```diff
@@ -266,6 +266,9 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
         if self.config.doWriteSubtractedExposure:
             outputExposure = inputs["inputExposure"].clone()
             outputExposure.image -= subtractor.image
+            if self.config.doApplySkyCorr and (inputs['skyCorr'] is not None):
```

Comment:
The quotes here should be changed from `'` to `"`, in keeping with the rest of the script.